testing/yuca/wrappers/test_coevolution_wrapper.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from  yuca.wrappers.coevolution_wrapper import CoevolutionWrapper
```

Comment:
I think that the development of the covariant matrix adaptation co-evolution strategy class, `CMACES`, makes the co-evolution wrapper redundant.